contributions/catalog/contribution_catalog.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
It is not important, but can we change the name of the file like the other BB? They use something like *_rest_service.py. So maybe this one becomes like catalog_rest_service.py? But it is not that important.